src/contributors/gilgold.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Gil Goldstein",
+    "desc": "Software Engineer at Indeed | Creator of Tabox for Chrome",
+    "github": "https://github.com/gilgold",
+    "twitter": "https://twitter.com/gilgoldstein",
+    "linkedin": "https://www.linkedin.com/in/gilgoldstein"
+}
```

Comment:
Delete this file. And don't Update unwanted files. This PR should focus on adding fonts.

Reply:
yeah sorry for that didn't see that . Made the changes